ksp_plugin_adapter/flight_planner.cs

```diff
@@ -347,47 +353,61 @@ class FlightPlanner : VesselSupervisedWindowRenderer {
     }
   }
 
-  private bool RenderAddManœuvreButton(int index) {
+  private bool RenderCoast(int index) {
     string vessel_guid = predicted_vessel.id.ToString();
-    if (UnityEngine.GUILayout.Button(
+    using (new UnityEngine.GUILayout.HorizontalScope()) {
+      double start_of_coast = index == 0
+          ? plugin.FlightPlanGetInitialTime(vessel_guid)
+          : burn_editors_[index - 1].final_time;
+      string coast_description = index == burn_editors_.Count
+          ? "Final trajectory"
+          : $@"Coast for {
+                (burn_editors_[index].initial_time -
+                 start_of_coast).FormatDuration(show_seconds : false)}";
+      UnityEngine.GUILayout.Label(
+          coast_description);
+      if (UnityEngine.GUILayout.Button(
             "Add manœuvre",
-            UnityEngine.GUILayout.ExpandWidth(true))) {
-      double initial_time;
-      if (index == 0) {
-        initial_time = plugin.CurrentTime() + 60;
-      } else {
-        initial_time = plugin.FlightPlanGetManoeuvre(
-                            vessel_guid,
-                            index - 1).final_time + 60;
-      }
-      var editor =
-          new BurnEditor(adapter_,
-                         predicted_vessel,
-                         initial_time,
-                         index,
-                         get_burn_at_index : burn_editors_.ElementAtOrDefault);
-      Burn candidate_burn = editor.Burn();
-      var status = plugin.FlightPlanInsert(
-          vessel_guid, candidate_burn, index);
-
-      // The previous call did not necessarily create a manœuvre.  Check if
-      // we need to add an editor.
-      int number_of_manœuvres =
-          plugin.FlightPlanNumberOfManoeuvres(vessel_guid);
-      if (number_of_manœuvres > burn_editors_.Count) {
-        editor.Reset(plugin.FlightPlanGetManoeuvre(
-            vessel_guid, index));
-        burn_editors_.Insert(index, editor);
-        UpdateBurnEditorIndices();
+            GUILayoutWidth(4))) {
+        double initial_time;
+        if (index == 0) {
+          initial_time = plugin.CurrentTime() + 60;
+        } else {
+          initial_time = plugin.FlightPlanGetManoeuvre(
+                              vessel_guid,
+                              index - 1).final_time + 60;
+        }
+        var editor = new BurnEditor(
+            adapter_,
+            predicted_vessel,
+            initial_time,
+            index,
+            get_burn_at_index : burn_editors_.ElementAtOrDefault);
+        editor.minimized = false;
```

Comment:
Could we just say that a burn editor is created maximized?  Shooting at the member variable is ugly.

Reply:
It is added maximized, but created minimized when loading an existing flight plan.

This is a property which is publicly write-only, not a member variable.